pkg/cli/credential.go

```diff
@@ -43,9 +43,11 @@ func (c *Credential) Run(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("failed to read CLI config: %w", err)
 	}
 
-	ctx := c.root.CredentialContext
+	ctxs := c.root.CredentialContext
 	if c.AllContexts {
-		ctx = credentials.AllCredentialContexts
+		ctxs = []string{credentials.AllCredentialContexts}
+	} else if len(ctxs) == 0 {
+		ctxs = []string{credentials.DefaultCredentialContext}
```

Comment:
I think it already is in the Complete function in the `gptscript/gptscript.go` file. But without it in places like this, the `c.root.CredentialContext` still ends up being empty when the user doesn't specify anything. Not sure why that's the case. Should I be manually calling the Complete function here?

Reply:
I'm surprised this isn't working as I expect, then.

Reply:
I took a longer look at this: instead of completing the entire `opts` (created on line 53), we are completing individual pieces of it (lines 55 and 56).

It would be better to complete everything instead of the individual pieces, but I won't hold up merging this based on that.

Reply:
Sweet, that worked. Thanks. I should have spent more time looking at this myself, lol